src/tests/server/database/user.test.js

```diff
@@ -1,92 +1,133 @@
 import {
   initTestDatabase,
   concludeTesting,
+  clearTestData,
   Models,
 } from "../../../server/database/index.js";
 import { ROOT_DIR } from "../../../helpers.js";
 import dotenv from "@dotenvx/dotenvx";
 const envPath = resolve(join(ROOT_DIR, `.env`));
 dotenv.config({ quiet: true, path: envPath });
 
+/**
+ * @typedef {Object} User
+ * @property {number} id
+ * @property {string} name
+ * @property {string} slug
+ * @property {string} created_at
+ * @property {string} enabled_at
+ * @property {string} bio
+ */
+
+/**
+ * Creates a user with a random, unique name
+ *
+ * @returns {User} the created user
+ */
+const createUser = () => {
+  const username = randomUUID();
+  const user = Models.User.create({ name: username });
+
+  return user;
+};
+
+/**
+ * Creates an admin user
+ *
+ * @returns {User} the admin user
+ */
+const createAdminUser = () => {
+  const user = createUser();
+
+  Models.Admin.create({ user_id: user.id });
+
+  return user;
+};
+
 describe(`user tests`, async () => {
   before(async () => await initTestDatabase());
+  afterEach(() => clearTestData());
   after(() => concludeTesting());
 
   /*
   export function processUserLogin(userObject) {
   */
 
   test(`deleteUser`, () => {
-    const user = Models.User.create({ name: `disposable` });
+    const user = createUser();
     User.deleteUser(user);
   });
 
   test(`enable/disable`, () => {
-    let user = User.getUser(`test-user`);
+    let user = createUser();
+    const username = user.name;
 
     User.enableUser(user);
-    user = User.getUser(`test-user`);
+    user = User.getUser(username);
     assert.notEqual(user.enabled_at, null);
 
     User.disableUser(user);
-    user = User.getUser(`test-user`);
+    user = User.getUser(username);
     assert.equal(user.enabled_at, null);
   });
 
   test(`getAllUsers`, () => {
+    createUser();
+    createUser();
     const users = User.getAllUsers();
     assert.equal(users.length, 2);
   });
 
   test(`userIsAdmin`, () => {
-    const admin = User.getUser(`test-admin`);
-    assert.equal(User.userIsAdmin(admin), true);
+    const admin = createAdminUser();
+    const user = createUser();
 
-    const user = User.getUser(`test-user`);
+    assert.equal(User.userIsAdmin(admin), true);
     assert.equal(User.userIsAdmin(user), false);
   });
 
   test(`getUserSettings`, () => {
-    const admin = User.getUser(`test-admin`);
+    const admin = createAdminUser();
     let settings = User.getUserSettings(admin);
     assert.deepEqual(settings, {
-      name: `test admin`,
+      name: admin.name,
       admin: true,
       enabled: true,
       suspended: false,
     });
 
-    const user = User.getUser(`test-user`);
-    const s = User.suspendUser(user, `why not`);
+    const user = createUser();
+    User.suspendUser(user, `why not`);
     User.disableUser(user);
     settings = User.getUserSettings(user);
     assert.deepEqual(settings, {
-      name: `test user`,
+      name: user.name,
```

Comment:
oh hello... this is the correct change, but I'm going to be removing all the code related to UserSettings as part of #198, because it's not actually used by anything and so it has no right to exist right now =)